nx.json

```diff
@@ -86,6 +80,12 @@
         "runtime": "yarn -v"
       }
     ],
-    "production": ["default", "!{projectRoot}/src/test-setup.[jt]s"]
+    "production": [
```

Comment:
We were missing a bit more granularity on our definition of "production" files, we can exclude spec files and test specific config files from consideration